api/comments/serializers.py

```diff
@@ -111,6 +113,8 @@ def update(self, comment, validated_data):
                     comment.edit(content, auth=auth, save=True)
                 except PermissionsError:
                     raise PermissionDenied('Not authorized to edit this comment.')
+                except ValidationValueError:
+                    raise ValidationError('Ensure this field has no more than 500 characters.')
```

Comment:
The docstrings over there suggest that the use of `ValidationError` elsewhere in this file is straining the design of the system.